plugins/org.jboss.tools.openshift.express.ui/src/org/jboss/tools/openshift/express/internal/ui/command/DeleteApplicationHandler.java

```diff
@@ -75,8 +75,7 @@ private IStatus deleteApplications(final List<IApplication> appsToDelete, Shell
 
 	private List<IUser> getUsers(List<IApplication> applications) {
 		List<IUser> users = new ArrayList<>();
-		if (applications == null
-				|| applications.size() == 0) {
+		if (applications.isEmpty()) {
```

Comment:
sonar reports this as an if-statement that always returns `true`. Indeed, it is a private method and  the check of `null` already exist in `execute` method.
What else must be mentioned is that sonar has a rule that an empty list must be returned from the method and not null. That's why it always expects that list is not null.